CONTRIBUTING.md

```diff
@@ -1,39 +1,97 @@
-# Contributing
+# Contributing Guide
 
-First, it is awesome that you are considering contributing to Kubescape! Contributing is important and fun and we welcome your efforts.
+* [Contributor Guide](#contributing-guide)
+   * [Ways to contribute](#ways-to-contribute)
+  * [Find an issue](#find-an-issue)
+  * [Working on issues](#working-on-issues)
+  * [Ask for help](#ask-for-help)
+  * [Build and test locally](#build-and-test-locally)
+  * [Pull request process](#pull-request-process)
+  * [Sign your commits](#sign-your-commits)
 
-When contributing, we categorize contributions into two:
-* Small code changes or fixes, whose scope is limited to documentation, minor fixes, development that involves no more than a file or two
-* Complex features and improvements, with potentially unlimited scope
 
-If you have a small change, feel free to fire up a Pull Request.
+Welcome! It's awesome that you are considering contributing to Kubescape! Contributing is important and we welcome your efforts. 💖
 
-When planning a bigger change, please first discuss the change you wish with one of the maintainers and the community. In this case, opening an issue in the most relevant repository and raising the improvement you want to add in Kubescape slack or community meeting is a great start!
+As you get started, you are in the best position to give us feedback on areas of
+our project that we need help with. 
 
-Getting a sign-off from a maintainer is important so the maintainers are able to help guide you and let you know if you are going in the right direction. It is always good have a common view with others so we don't need to upset you later if we see the world in another way...
+If anything doesn't make sense, or doesn't work when you run it, please open a
+bug report and let us know!
 
-## Community
+### Code of conduct
 
-Read about where to find Kubescape community [here](COMMUNITY.md)
+Please follow our [code of conduct](CODE_OF_CONDUCT.md) in all of your interactions within the project.
 
-## Code of Conduct
+## Ways to contribute
 
-Please follow our [code of conduct](CODE_OF_CONDUCT.md) in all of your interactions within the project.
+We welcome many different types of contributions including:
+
+* New features
+* Builds, CI/CD
+* Bug fixes
+* Documentation
+* Issue Triage
+* Answering questions on Slack/Mailing List
+* Web design
+* Communications / Social Media / Blog Posts
+* Release management
+
+<!--Ben/Rotem/Matthias this language is from the CNCF suggested template, it incudes much more than we had in the original file, please indicate if something should be removed-->
```

Comment:
I am ok with this, we should include these things from the CNCF template (except the one bellow I noted)